### PR TITLE
Add flake8 config and trim code

### PR DESCRIPTION
**Pull request by ss77995ss**
### Description
1. Add `flake8` config file
2. Trim code in `split_sentences.py`
3. Ignore `.vscode`

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
+.vscode
```
Comment (ss77995ss) [inline .gitignore]: Sorry, I gave a wrong information about this yesterday :(

File: utils/split_sentence.py [hunk 2]
```diff
@@ -30,7 +30,7 @@ def get_sheet():
             creds = pickle.load(token)
     # If there are no (valid) credentials available, let the user log in.
     if not creds or not creds.valid:
-        if creds and creds.expired and creds.refresh_token:
+        if creds.expired and creds.refresh_token:
```
Comment (ss77995ss) [inline utils/split_sentence.py]: Not sure how python works, but what I concern is that we have already check `if not creds` condition, so we might not need to have condition `if creds`

File: utils/split_sentence.py [hunk 3]
```diff
@@ -76,13 +76,15 @@ def main():
         "values": word_sentence_list
     }
 
-    request = sheet.values().update(spreadsheetId=SPREADSHEET_ID, range=updated_range,
-                                    valueInputOption=value_input_option, body=value_range_body)
+    request = sheet.values().update(spreadsheetId=SPREADSHEET_ID,
```
Comment (ss77995ss) [inline utils/split_sentence.py]: This part just my preference 😆Not really need to change this line

File: .gitignore (hunk 1 above)
Comment (jimmYA-1995) [reply .gitignore]: That's fine. Let's add `.vscode/` in `.gitignore`.

File: utils/split_sentence.py (hunk 2 above)
Comment (ss77995ss) [reply utils/split_sentence.py]: OK, I see, I missed `OR` in the condition 😞, thanks!

**Pull request merged**